app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
Do not repeat abstract methods from a parent if you have an abstract class that inherits another abstract class.
So `Player` already child of `ABC` class , so you can interhance just from `Player`